Lib/ensurepip/__init__.py

```diff
@@ -164,9 +163,10 @@ def _bootstrap(*, root=None, upgrade=False, user=False,
         for name, package in _get_packages().items():
             if package.wheel_name:
                 # Use bundled wheel package
-                from ensurepip import _bundled
                 wheel_name = package.wheel_name
-                whl = resources.read_binary(_bundled, wheel_name)
+                wheel_path = resources.files("ensurepip") / "_bundled" / wheel_name
+                with wheel_path.open("rb") as fp:
+                    whl = fp.read()
```

Comment:
As FFY00 noted, `read_bytes` is in fact part of the Traversable protocol (with a default implementation).

```suggestion
                whl = wheel_path.read_bytes()
```